engine/src/main/java/org/terasology/persistence/serializers/ProtobufSerializer.java

```diff
@@ -30,38 +30,76 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
-// TODO: Document
 public class ProtobufSerializer {
+	/**
+	 * Converts the object into an array of bytes
+	 * @param object the object to be serialized
+	 * @param typeHandler contains how the object will be serialized
+	 * @return the byte array of the object
+	 * @throws IOException throws if there is an error writing to the stream
+	 */
     public <T> byte[] toBytes(T object, TypeHandler<T> typeHandler) throws IOException {
         try (ByteArrayOutputStream stream = new ByteArrayOutputStream()) {
             writeBytes(object, typeHandler, stream);
             return stream.toByteArray();
         }
     }
 
+    /**
+     * Writes an object's bytes to a file
```

Comment:
When an overloaded method like this solely forwards execution to another variant it helps to explain the subtle difference rather than nearly have identical javadoc blocks. You can use tags in Javadoc like `@see` to direct to other variants for more details then use one of them (one that doesn't just call another variant) to hold a more description variant of the javadoc as well